### DISCUSSION TITLE
間幕用のトークデータをadminから追加できるようにする

### DISCUSSION THREAD

**Discussion by takaishi** [cloudnativedaysjp/dreamkast#1372] [Ideas]
今はDBを直接編集するかRakeタスクで追加する必要があり、不便。

**Comment by takaishi** [comment]
https://github.com/cloudnativedaysjp/dreamkast/issues/713 でスポンサーセッションの話が出てきていて、トークの種別を識別できるようにする必要がありそう。

**Comment by takaishi** [comment]
https://event.cloudnativedays.jp/cicd2021/admin/tracks に追加用のフォームへのリンクを貼る。 `日付切り替えタブの` の上側あたりかな。

![image](https://user-images.githubusercontent.com/107187/133867395-b0713d52-12e8-47ab-9d92-9f215b78f70b.png)

フォームでは複数レコードを追加・編集できるようにしたい。また、入力値やデフォルト値は以下の通り。

入力可能フィールド

- title
- conference_day_id
- start_time
- end_time
- track_id

デフォルト値

- conference_id
- show_on_timetable (false)
- video_published (false)

**Comment by takaishi** [comment]
時間間違えた時とかのことを考えるとフォームから登録するのもなんか微妙な気がしてきた…

**Comment by takaishi** [comment]
ちょっといい方法が思いつかない & 他に優先順位が高いIssueがあるので一旦SprintBacklogからは外す

**Comment by TakumaNakagame** [comment]
Broadcastの振り返りで取り上げられましたが、本件の優先度を上げてほしいです。

というのも、現在はBroadcastの多くの仕組みがDkのAPIなどの情報を参照しています。
Broadcast的には、休憩なども含めた最終版の情報を出来るだけ早めに（開催2週間前くらい）にあると助かります。:bow:
